ci/fedora.sh

```diff
@@ -4,13 +4,14 @@ set -e
 
 DNFCMD="dnf"
 SPECFLAG="--spec "
+#ARCH="$(uname -m)"
 
-if [ "$OSVERSION" = "rawhide" ] || [ "$((OSVERSION+0))" -gt 40 ]; then
-    DNFCMD="dnf5"
-    SPECFLAG=""
+if [ "$OSVERSION" = "rawhide" ] || [ "$((OSVERSION + 0))" -gt 40 ]; then
+	DNFCMD="dnf5"
+	SPECFLAG=""
 fi
 
-$DNFCMD install -y rpm-build make gcc gcc-c++ libdrm-devel systemd-rpm-macros glibc-devel.x86_64 wget git "$DNFCMD-command(builddep)"
+$DNFCMD install -y rpm-build make gcc gcc-c++ libdrm-devel systemd-rpm-macros glibc-devel wget git "$DNFCMD-command(builddep)"
```

Comment:
The `.x86_64` was there because sometimes if glibc-devel was installed for i686 it would not trigger the install for x86_64, but I think this should be rare in real-world scenarios, but it may depend on what people do with their machines.

For GitHub releases it should be alright, but maybe add a comment on the codebase with this small piece of trivia :)

Reply:
thanks for the review, I have made the suggested change.